java/org/apache/jasper/EmbeddedServletOptions.java

```diff
@@ -635,6 +648,19 @@ public EmbeddedServletOptions(ServletConfig config, ServletContext context) {
             }
         }
 
+        String jspWhiteSpaceTrim = config.getInitParameter("JSPWhiteSpaceTrimming");
+        if (jspWhiteSpaceTrim != null) {
+            if (jspWhiteSpaceTrim.equalsIgnoreCase("true")) {
+                this.JSPWhiteSpaceTrimming  = true;
+            } else if (jspWhiteSpaceTrim.equalsIgnoreCase("false")) {
+                this.JSPWhiteSpaceTrimming  = false;
+            } else {
+                if (log.isWarnEnabled()) {
+                    log.warn(Localizer.getMessage("Invalid Value for the flag"));
```

Comment:
The `key` for `getMessage()` looks weird here.
See the other usages of Localizer.getMessage() with `jsp.warning.**` keys.